transcendence-app/src/chat/chatroom/infrastructure/db/postgres/chatroom.postgres.repository.ts

```diff
@@ -23,9 +22,10 @@ export class ChatroomPostgresRepository
     super(pool, table.CHATROOM, Chatroom);
   }
   async addChatroom(chatroom: Partial<Chatroom>): Promise<Chatroom | null> {
-    const chatroomData = await this.pool.transaction<Chatroom>(
+    const chatroomData = await makeTransactionalQuery<Chatroom>(
+      this.pool,
       async (client) => {
-        const chatroomData = await insertWithClient(
+        const chatroomData = await BasePostgresRepository.insertWithClient(
```

Comment:
suggestion (see comments below):
```suggestion
        const chatroomData = await this.insertWithClient(
```